### ISSUE TITLE
`Could not find a declaration file for module 'vue-countup-v3'`

### ISSUE THREAD

**Issue by KnifeFed** [jizai1125/vue-countup-v3#11]
I'm using it with Nuxt 3 and when starting the dev server, vue-tsc throws this error:

```
Could not find a declaration file for module 'vue-countup-v3'. '/<project_dir>/node_modules/.pnpm/vue-countup-v3@1.3.0_vue@3.3.4/node_modules/vue-countup-v3/dist/vue-countup-v3.es.js' implicitly has an 'any' type.
  There are types at '/<project_dir>/node_modules/vue-countup-v3/dist/countup.vue.d.ts', but this result could not be resolved when respecting package.json "exports". The 'vue-countup-v3' library may need to update its package.json or typings.
```

Editing the `package.json` in node_modules and moving [this](https://github.com/jizai1125/vue-countup-v3/blob/main/package.json#L24) to [here](https://github.com/jizai1125/vue-countup-v3/blob/main/package.json#L20-L21) got rid of the error message for me.

**Comment by jizai1125** [comment]
Can you provide a reproduction? I haven't found any issues.

**Comment by KnifeFed** [comment]
Turns out it's fixed in Nuxt 3.6.3, which then caused this issue: https://github.com/nuxt/nuxt/issues/22156
Fixed by upgrading from Nuxt 3.6.2 and removing those lines from `nuxt.config.ts`.
